src/models/Users.php

```diff
@@ -148,7 +148,7 @@ public function createOne(
         $req->bindParam(':firstname', $firstname);
         $req->bindParam(':lastname', $lastname);
         $req->bindParam(':register_date', $registerDate);
-        $req->bindParam(':validated', $isValidated, PDO::PARAM_INT);
```

Comment:
This should not be necessary but yet here we are.
https://app.circleci.com/pipelines/github/elabftw/elabftw/7776/workflows/df7d7713-65c6-4507-92ba-6b2c0e65b426/jobs/11280/parallel-runs/0/steps/0-116